lib/shopify-cli/extension.rb

```diff
@@ -0,0 +1,4 @@
+module ShopifyCli
+  class Extension
```

Comment:
This doesn't seem to be used anywhere?

Reply:
Oh you are right, this is actualy a miss-commit and part of other experimentation that I was doing. Thanks for catching that!